pkg/client/errors.go

```diff
@@ -24,12 +24,25 @@ import (
 
 // Errors related to Client connection and health check
 var (
+
+	// ErrIllegalArguments indicates illegal values of function argument
```

Comment:
ErrIllegalArguments indicates illegal arguments provided to a method? wdyt?

Reply:
Updated in 3b17be074cc700645a5d1888d152b0f76babe937